### ISSUE TITLE
Postman runner continues when browser is closed before choosing to "Stop Run"

### ISSUE THREAD

**Issue by JamesMay6** [a85/POSTMan-Chrome-Extension#10430]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the Issue

If a collection is running in Postman runner which is designed to loop, even when a brower is refreshed or accidenatally closed prior to choosing to "stop Run", the requests continue - i can evidence this as the actions my apis are doing are continue to perform despite not taking place on the runner. 

How do i interupt these runs and STOP them!?

### Steps To Reproduce

Create API collection
Ensure API collection is designed to call another api in the suite e.g.  API 1 & API 2 - API 1 calls API2 and API 2 call API 1 
Start the API's in the "Runner"
Closer the browser (DO NOT PRESS STOP RUN)
Apis continue to process

### Environment Information

```markdown
- Operating System: Windows 10
- Platform Type: Chrome 95
- Postman Version: Version 9.0.9-211103-1300
- Using Postman Desktop Agent
```

**Comment by JamesMay6** [comment]
Note: i have found that quitting the postman desktop agent stops any runs from continuing. i have not tried this with the cloud service - it may be a specific issue with the postman agent.

**Comment by malvikach** [comment]
@JamesMay6 Thanks for writing to us and letting us know about this issue. Our team will be looking into this.

**Comment by malvikach** [comment]
Hi @JamesMay6 👋 

The above has been fixed in the latest Postman version (10.6.*).  If you accidentally close the tab now, you will be asked to confirm if you wish to stop the run or continue it.

![Screenshot 2022-12-08 at 11 08 02 PM](https://user-images.githubusercontent.com/89449769/206524525-8555fb8c-e5ac-4c07-a01c-740eb5a3b22c.png)

Hope this helps in taking an informed decision about the run execution. 

I am closing this issue, please feel free to let us know if you have any other feedback. Thanks again for bringing this to our notice. 

Cheers!
Malvika
